matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -430,6 +443,23 @@ impl WebRtcSocket {
 }
 
 impl<C: ChannelPlurality> WebRtcSocket<C> {
+    /// Close this socket, disconnecting all channels.
+    pub fn close(mut self) {
```

Comment:
Should we have a `close_channel` and a close_socket`?

Reply:
Do you mean you would like the `close` methods on `WebRtcChannel` and `WebRtcSocket` to be renamed to `close_channel` and `close_socket` respectively?